.travis.yml

```diff
@@ -179,7 +179,7 @@ jobs:
       install:
         - cmake -H. -Bbuild '-DCMAKE_BUILD_TYPE=Release' '-DCMAKE_CXX_COMPILER=g++-5'
         - cmake --build build -- -j4
```

Comment:
Have you tried to build it with `-j2` or `-j3` it may be quicker on Travis 2 core machines?

Reply:
I haven't played with the build yet , though I was wondering about the choice of -j4 too, considering the machines we run the builds on. It may be that we'll benefit from different choices on different platforms here, but I'll do some experiments under a different PR.